cg/apps/tb/dto/summary_response.py

```diff
@@ -0,0 +1,14 @@
+from pydantic import BaseModel
+
+
+class Summary(BaseModel):
+    order_id: int
+    total: int
```

Comment:
Should we skip this in the response from trailblazer and just use the case count as the total instead?

Reply:
Yeah, I think so! And using the case count will ensure we get an accurate count despite the analyses not having been created yet in trailblazer